CMakeLists.txt

```diff
@@ -27,8 +27,11 @@ INCLUDE(wawaka_common)
 LIST(APPEND WASM_LIBRARIES ${WW_COMMON_LIB})
 LIST(APPEND WASM_INCLUDES ${WW_COMMON_INCLUDES})
 
+SET(CONTRACT_FAMILIES exchange-contract digital-asset-contract inference-contract)
+INCLUDE(Local.cmake OPTIONAL)
+
 ENABLE_TESTING()
 
```

Comment:
Can we please update the install.md with details on how to use the Local.cmake feature introduced in this PR? I see that you have added details to the PR header, but a new reader is unlikely to use this unless we have it as part of install.md.

Reply:
i would prefer to just add a comment in the makefile